pkg/scheduler/preemption/preemption_test.go

```diff
@@ -1879,10 +1857,9 @@ func TestPreemption(t *testing.T) {
 				t.Errorf("Reported %d preemptions, want %d", preempted, tc.wantPreempted.Len())
 			}
 
-			beforeResourceNodes := resourceNodes(beforeSnapshot)
-			afterResourceNodes := resourceNodes(snapshotWorkingCopy)
-			if diff := cmp.Diff(beforeResourceNodes, afterResourceNodes, snapCmpOpts); diff != "" {
-				t.Errorf("Snapshot was modified (-initial,+end):\n%s", diff)
+			snapshotComparer := NewSnapshotComparer(cmpopts.IgnoreFields(cache.ResourceNode{}, "Usage"))
```

Comment:
```suggestion
			snapshotComparer := NewSnapshotComparer()
```
Is it working here without `IgnoreFields `?

Reply:
yes

Reply:
yep